core-errors/src/lib.rs

```diff
@@ -220,6 +220,15 @@ pub enum ExecutionError {
     /// An error occurs in attempt to parse invalid string in `gr_debug` sys-call.
     #[display(fmt = "Invalid debug string passed in `gr_debug` sys-call")]
     InvalidDebugString = 0,
+
+    /// The error occurs when program's balance is less than rent it tries to pay.
+    #[display(fmt = "Existing value {value_left} is not enough to pay rent {rent}")]
+    NotEnoughValue {
```

Comment:
Isn't it too general? Will it be better `NotEnoughValueForRent`?

Reply:
agree